build.xml

```diff
@@ -45,7 +45,7 @@
                         <include name="**/*.jar" /> <exclude name="**/javax.inject*.jar"/>
 		</fileset>
 		<fileset dir="${tck.home}">
-                        <include name="jakarta.inject-tck-1.0.jar" />
+                        <include name="jakarta.inject-tck-2.0.0.RC3.jar" />
```

Comment:
Not sure if it is easy to parameterise this value. Will it not be required to change to final jar name later ?

Reply:
removed version